pvacseq/lib/main.py

```diff
@@ -12,7 +12,25 @@
     import lib
 from lib.prediction_class import *
 from lib.pipeline import *
+from lib.config_files import additional_input_file_list_options
+
 import shutil
+import yaml
+
+def parse_additional_input_file_list(additional_input_file_list):
+    if additional_input_file_list:
+        with open(additional_input_file_list, 'r') as stream:
+            try:
+                additional_input_files = yaml.load(stream)
+            except yaml.YAMLError as exc:
```

Comment:
If I'm reasoning about this correctly, failure to parse the YAML file will result in an exception which is caught here. This will result in a value of `None` (I think) being returned from this function, which would be the same as if they hadn't specified a YAML file at all (excepting the error message printed to the console) and execution will continue as normal.

Is that the desired behavior?